cfn/cfn.json

```diff
@@ -0,0 +1,375 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Metadata": {
+        "AWS::CloudFormation::Designer": {
```

Comment:
> i like to be able to sanity-check the template in the designer UI

its possible to open a stripped down template in the designer on-demand, which I think helps here?